src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -246,6 +246,23 @@ void main() {
 }
 ```
 
+By default, `flutter_driver` will wait until there are no pending frames,
+and thus tests similar to the example above will fail with a timeout if,
```

Comment:
and thus tests similar to the example above will fail with a timeout if, => and tests similar to the example above fail with a timeout if,